src/Core/Settings.h

```diff
@@ -641,10 +641,17 @@ class IColumn;
 // End of FORMAT_FACTORY_SETTINGS
 // Please add settings non-related to formats into the COMMON_SETTINGS above.
 
+// settings work in query runtime
+#define RUNTIME_QUERY_SETTINGS(M) \
+    M(Bool, is_reinterpreted_execution, false, "Queries such as show tables will be reinterpreted to select query.", 0)
+// End of RUNTIME_QUERY_SETTINGS
```

Comment:
This new setting looks too complex for this easy task, the previous solution with

```
bool ignoreQuota() const override { return false; }
```

looked more clear and obvious to me.

Reply:
ok